roles/geniza_setup/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Install local settings for flask
+  template:
+    src: "local_settings.cfg.j2"
+    dest: "{{ deploy }}/scripts/local_settings.cfg"
+    owner: "{{ deploy_user }}"
+    group: "{{ deploy_user }}"
+
+- name: Ensure bin directory exists.
+  file:
+    path: "/home/{{ deploy_user }}/bin"
+    state: directory
+    mode: 0700
+    owner: "{{ deploy_user }}"
+    group: "{{ deploy_user }}"
```

Comment:
I can't remember why this is here! I think maybe it is a leftover from the cron scripts, which I may have first included here before I generalized for re-use with S&co. I think you're right that we don't need it.  It would be great if we didn't need a separate geniza setup role and could fold the local settings handling into an existing task.

Reply:
I just checked — I think you're right, it looks like it should work to switch to using the existing `install_local_settings` role. Let's try that and get rid of this one!